object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -724,6 +640,10 @@ private void handleSuccessorBlocks(
     }
   }
 
+  private boolean isTempVar(LocalVariable localVariable) {
+    return localVariable.getElement().getSimpleName().toString().contains("temp-var");
```

Comment:
can't we add a method to the type factory to check this properly by checking for membership in the bi-map?

Reply:
I removed this method because I don't use it.